tools/scripts/AksEdgeQuickStart/AksEdgeQuickStartForAio.ps1

```diff
@@ -0,0 +1,264 @@
+<#
+  QuickStart script for setting up Azure for AKS Edge Essentials and deploying the same on the Windows device
+#>
+param(
+    [ValidateNotNullOrEmpty()]
+    [String] $SubscriptionId,
+    [ValidateNotNullOrEmpty()]
+    [String] $TenantId,
+    [ValidateNotNullOrEmpty()]
+    [String] $Location,
+    [Switch] $UseK8s,
+    [string] $Tag
+)
+#Requires -RunAsAdministrator
+New-Variable -Name gAksEdgeQuickStartVersion -Value "1.0.231016.1400" -Option Constant -ErrorAction SilentlyContinue
+
+New-Variable -Option Constant -ErrorAction SilentlyContinue -Name arcLocations -Value @(
```

Comment:
Can we check with AIO team to confirm all these locations will be supported with AIO?

Reply:
Sure, let me check on the location support for AIO.